docs/api/system-preferences.md

```diff
@@ -450,6 +450,12 @@ Returns `Promise<Boolean>` - A promise that resolves with `true` if consent was
 
 This user consent was not required until macOS 10.14 Mojave, so this method will always return `true` if your system is running 10.13 High Sierra or lower.
 
+### `systemPreferences.isScreenCaptureAllowed()` _macOS_
```

Comment:
I think it'd be worth adding a link to this method from `systemPreferences.getMediaAccessStatus` and `desktopCapturer` to make things a little easier for folks and promote a proactive approach to solving this inevitable problem.